src/sandbox/components/Grid.svelte

```diff
@@ -61,179 +92,138 @@
       width: ${(colEnd - colStart + 1) * columnWidth + 2}px;
     `
   })(copySelection);
+  $: if (selection) requestAnimationFrame(() => scrollIntoView(selection));
+  $: if (highlight) requestAnimationFrame(() => scrollIntoView(highlight));
```

Comment:
I can imagine the first statement should not execute if highlight is present. So might be if (highlight) {} else {} is better?

Reply:
Perhaps this would be even better?

```suggestion
  $: requestAnimationFrame(() => scrollIntoView(isEditMode? highlight : selection));
```